tui/installer/screens.py

```diff
@@ -178,6 +178,26 @@ def overwrite_warning(answers):
     if button == 'back': return LEFT_BACKWARDS
     return RIGHT_FORWARDS
 
+def bios_warning(answers):
+    text = ("Booting %s hosts in BIOS mode is now deprecated. "
+            "You can still install your %s hosts in BIOS boot mode. "
+            "However, doing so can prevent you from upgrading your %s "
```

Comment:
Perhaps 'may prevent' rather than 'can prevent'?

Reply:
This text more or less came from docs, but that sounds better to me so I'll change it.